src/components/Map.tsx

```diff
@@ -66,22 +73,44 @@ const Map = ({
   interactive = true,
   onFeatureSelected = () => {},
   showNoLocation,
+  showNearbyHotspots = false,
   ...props
 }: Props) => {
+  const dispatch = useAppDispatch()
   const colors = useColors()
   const { t } = useTranslation()
   const map = useRef<MapboxGL.MapView>(null)
   const camera = useRef<MapboxGL.Camera>(null)
   const [loaded, setLoaded] = useState(false)
   const [userCoords, setUserCoords] = useState({ latitude: 0, longitude: 0 })
   const [centerOffset, setCenterOffset] = useState(0)
+  const styles = useMemo(() => makeStyles(colors), [colors])
+
+  const {
```

Comment:
Seeing as though this is a pretty lightweight object, I'd recommend using lodash's `isEqual` function here
```ts
 const networkHotspots = useSelector(
    (state: RootState) => state.networkHotspots.networkHotspots,
    isEqual,
  )
```